pages/code_schools.js

```diff
@@ -232,6 +232,7 @@ function CodeSchools({ allSchools }) {
               placeholder="Start typing a state..."
               className={styles.select}
               isMulti
+              closeMenuOnSelect={false}
```

Comment:
Why is this necessary? If `isMulti` is true, shouldn't it propagate down to ThemedReactSelect?